core/kazoo/include/kz_config.hrl

```diff
@@ -0,0 +1,6 @@
+-ifndef(KZ_CONFIG_HRL).
+
+-define(FEDERATE_BINDING, case kz_config:get_boolean('amqp', 'use_federation') of 'true' -> ['federate']; _ -> [] end).
```

Comment:
@jamhed howdy, two notes.  

1) I would expect this parameter to live in the system_config database but (unless I am mistaken) I think you are pulling it from the ini file.  
2) This parameter appears to enable/disable federation globally by its naming but it should be specific to the notifications.  In system_config we have a document (notify) for the notify app and another notifications for the teletype app.  I would be ok with either using one parameter for both (in which case it should be on the notification document) or on parameter per app.  In this case since the context would be on the system_config/notification document then 'use_federation' would be an acceptable parameter name.

If you want we can discuss this further in the standup tomorrow!